Documentation/Configuration/GlobalVariables.rst

```diff
@@ -57,6 +57,18 @@ $GLOBALS
 
    Contains information related to BE skinning.
 
+.. confval:: TSFE
+
+   :Path: $GLOBALS
+   :type: TypoScriptFrontendController
+   :Defined: :file:`typo3/sysext/core/ext_tables.php`
+   :Frontend: yes
+
+   Contains an instantiation of :php:`TypoScriptFrontendController`. Provides
```

Comment:
Please add a hint that the global variable should be only used as a last resort (when no other option is possible).

Reply:
Added to global vars page and TSFE page.